src/OpenTK/Platform/Windows/WinGLNative.cs

```diff
@@ -735,7 +735,7 @@ private IntPtr WindowProcedure(IntPtr handle, WindowMessage message, IntPtr wPar
                     break;
 
                 case WindowMessage.ERASEBKGND:
-                    return new IntPtr(1);
+                    break;
```

Comment:
Once OpenGL is running this is a waste getting Windows to clear the background for us. Can this be turned off once the main loop is running.